src/Compilers/Core/Portable/InternalUtilities/JsonWriter.cs

```diff
@@ -163,16 +165,17 @@ public void Dispose()
         //
         // https://github.com/dotnet/corefx/blob/master/src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/JavaScriptString.cs
         //
-        private static string EscapeString(string value)
+        private static string EscapeString(string? value)
         {
-            PooledStringBuilder pooledBuilder = null;
-            StringBuilder b = null;
+            PooledStringBuilder? pooledBuilder = null;
+            StringBuilder? b = null;
 
-            if (string.IsNullOrEmpty(value))
+            if (RoslynString.IsNullOrEmpty(value))
             {
                 return string.Empty;
             }
 
+#nullable disable // Needs work...
```

Comment:
>// Needs work... [](start = 18, length = 16)

Tracking?

Reply:
➡️ Better. I rewrote the method to highlight how it was already null-safe. 😄